manifests/kiali-ossm/manifests/kiali.clusterserviceversion.yaml

```diff
@@ -27,7 +27,7 @@ metadata:
     containerImage: ${KIALI_OPERATOR_REGISTRY}
     capabilities: Deep Insights
     support: Red Hat
-    description: "Kiali project provides answers to the questions: What microservices are part of my Istio service mesh and how are they connected?"
+    description: "This productized operator provides Kiali and OSSMC.  Kiali is the Istio observability and management Console.  OSSMC is the OpenShift Service Mesh Console plugin, powered by Kiali."
```

Comment:
your old-school "two spaces after a period" bugs me... we aren't using typewriters anymore. But, be that as it may, I'll let you keep it since you insist on thinking that's how today's generation writes. :p

Reply:
Hold on while I get my White-Out.